### PR TITLE
New /new page.

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

Add English only long form variant of  /new for a/b testing. (Traffic cop config will be submitted in a follow up PR)

## Issue / Bugzilla link

See #9019

## Testing

http://localhost:8000/en-US/firefox/new/?v=b and behind the switch `desktop-download`

### HTML

- buttons & links have analytics attributes
- links appear to go to their intended places

### CSS

- This page will need to provide an enhanced experience to older browsers but we can exclude old browsers from the a/b test.
- RTL is also excluded from the a/b test.

### JS

- Browser comparison chart should show user's current browser if it is Chrome, Edge, or Safari, and fall back to their OS's default browser if their current browser is not in that list
- Protection report button should only appear for Firefox 70 and higher, should open protection report
- Mobile banner should display in place of the hero banner on android and ios
- System requirements should link to the user's platform if it's windows, macos, or linux
- Elements with `js-animate` should animate when scrolled into view on browsers supporting CSS animation and intersection observer, and should still be visible without support. 
  - See any performance no-nos in this function? I'm worried about the setTimeout
  - They should only animate when scrolled up into view, not down, and only the first time (this is easiest to test with a really narrow viewport)

### CODE REVIEW

**Comment by stephaniehobson** [comment]
@Errolyn I have cleanup to do on the CSS and images. But, it's a big PR, if you could review the Javascript while I work on that it'd speed things up. TY!

**Comment by stephaniehobson** [comment]
Oop, should have removed the DNM when I added the comparison table stuff.

**Comment by alexgibson** [comment]
Friendly reminder that if this is a traffic cop experiment, you may need to make sure this new template doesn't cause any existing integration tests to fail, and consider temporarily skipping ones that are effected.

File: media/css/firefox/new/desktop/download.scss [hunk 1]
```diff
@@ -0,0 +1,1448 @@
...
+
+    @media #{$mq-lg} {
+        padding-left: $layout-xl;
+        padding-right: $layout-xl;
+    }
+
+    img {
+        display: block;
+        margin: 0 auto;
+        background-color: #303035;
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply media/css/firefox/new/desktop/download.scss:1206]: This one is sampled from the image so I'd like to keep the close colour match.

File: media/css/firefox/new/desktop/download.scss [hunk 2]
```diff
@@ -0,0 +1,1448 @@
...
+
+// button container
+
+.mzp-c-button-download-container {
+    main & { // target main to avoid conflicting with nav
+        margin-bottom: 0;
+    }
+
+    .t-intro & {
+        text-align: left;
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply media/css/firefox/new/desktop/download.scss:99]: I'll fix RTL when I localize the page.

File: bedrock/firefox/templates/firefox/new/desktop/download.html [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+
+  <section class="t-highlights">
+    <h2 class="c-section-title mzp-has-zap-11">Do what you do online.<br> Firefox Browser <strong>isn’t</strong> watching.</h2>
+
+    <div class="t-compare c-block">
+      <div class="c-block-container">
+        <div class="c-block-body l-v-center l-h-end">
+          <h3 class="mzp-u-title-sm">How Firefox compares to other browsers</h3>
+          <p>Get all the speed and tools with none of the invasions of privacy. Firefox Browser collects so little data about you, we don’t even require your email address to download. That’s because unlike other browsers, we have no financial stake in following you around the web.</p>
+          <!-- <p><a href="" class="mzp-c-cta-link">How we compare to other browsers</a></p> -->
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/new/desktop/download.html:97]: Seems like these are still in a bit of flux?

**Comment by stephaniehobson** [comment]
Updated with tokens and locale independent links.

**Pull request merged**